Dockerfile

```diff
@@ -21,6 +21,9 @@ COPY --from=builder \
   /go/src/github.com/smartcontractkit/chainlink/gui/dist \
   /go/src/github.com/smartcontractkit/chainlink/gui/dist
 
+# Disable TLS... for now
+ENV CHAINLINK_DEV true
```

Comment:
Do we want to default to dev mode? Could we just change `cldev` and any other consumer of our chainlink docker image to explicitly set `ENV CHAINLINK_DEV true` via command line?

This is already being done in `internal/ci/ethereum_test`, perhaps all that's left is updating `cldev`.

Reply:
This allows these images to work with our current automation, which does not ship any TLS key material.

Reply:
Can we modify our current automation to set `CHAINLINK_DEV true`? By automation do you mean `codeship` and `kubernetes`?

Reply:
I'm not strongly attached to this issue. But it seems counterintuitive to have our program default to TLS, but our docker image default to no TLS. So I'm hoping we can explicitly set `CHAINLINK_DEV true` wherever we consume the docker image via command line or environment variables.

Reply:
Done, see https://github.com/smartcontractkit/devnet/pull/18